bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/config/config2.xml

```diff
@@ -96,10 +96,6 @@
 			<description>@text/thing-type.config.shelly.deviceIp.description</description>
 			<context>network-address</context>
 		</parameter>
-		<parameter name="userId" type="text" required="false">
-			<label>@text/thing-type.config.shelly.userId.label</label>
-			<description>@text/thing-type.config.shelly.userId.description</description>
-		</parameter>
```

Comment:
Why is this removed - no user name, but still a password?

Reply:
correct, Gen2 devices hace a fixed user id "admin"